examples/emoji_pet.py

```diff
@@ -0,0 +1,77 @@
+"""
+    emoji_pet.py
+    ~~~~~~~~
+
+    A reactive "pet" which changes face depending on certain inputs
+    editable by the user. Emotions are functions of an animation function.
+    You can optionally get reactive sounds from by
+    connecting a speaker/buzzer/headphone to P0 and GND.
+"""
+from microbit import *
+import music
+
+# Various emoji states of the face
+neutral_face = Image("09090:00000:99999:00000:00000")
+neutral_face2 = Image("09090:00000:00000:99999:00000")
+
+squiggle_face1 = Image("09090:00000:90000:99999:00009")
+squiggle_face2 = Image("09090:00000:00009:99999:90000")
+
+grin_face = Image("09090:00000:90009:09990:00000")
+happy_face = Image("09090:00000:99999:90009:09990")
+
+frowning_face = Image("09090:00000:09990:90009:00000")
+sad_face = Image("09090:00000:09990:90009:90009")
+
+small_tongue = Image("07090:00000:99999:00990:00000")
+
+
+def animate(faces, delay=500):
+    # A function to show images one after the other
+    for face in faces:
+        display.show(face)
+        sleep(delay)
+
+
+def get_happy(loops=2):
+    # A series of happy faces
+    music.play(music.BA_DING, wait=False)
+    for i in range(loops):
+        animate([neutral_face, grin_face, happy_face])
```

Comment:
You can replace this with `display.show([neutral_face, grin_face, happy_face], delay = 500)`